halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/openstack/OpenstackEditAccountCommand.java

```diff
@@ -0,0 +1,138 @@
+package com.netflix.spinnaker.halyard.cli.command.v1.config.providers.openstack;
+
+import com.beust.jcommander.Parameter;
+import com.netflix.spinnaker.halyard.cli.command.v1.config.providers.account.AbstractEditAccountCommand;
+import com.netflix.spinnaker.halyard.config.model.v1.node.Account;
+import com.netflix.spinnaker.halyard.config.model.v1.providers.openstack.OpenstackAccount;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class OpenstackEditAccountCommand extends AbstractEditAccountCommand<OpenstackAccount> {
+  protected String getProviderName() {
+    return "openstack";
+  }
+
+  @Parameter(
+      names = "--auth-url",
+      description = OpenstackCommandProperties.AUTH_URL_DESCRIPTION
+  )
+  private String authUrl;
+
+  @Parameter(
+      names = "--username",
+      description = OpenstackCommandProperties.USERNAME_DESCRIPTION
+  )
+  private String username;
+
+  @Parameter(
+      names = "--password",
+      description = OpenstackCommandProperties.PASSWORD_DESCRIPTION
+  )
+  private String password;
+
+  @Parameter(
+      names = "--project-name",
+      description = OpenstackCommandProperties.PROJECT_NAME_DESCRIPTION
+  )
+  private String projectName;
+
+  @Parameter(
+      names = "--domain-name",
+      description = OpenstackCommandProperties.DOMAIN_NAME_DESCRIPTION
+  )
+  private String domainName;
+
+  @Parameter(
+      names = "--regions",
+      variableArity = true,
+      description = OpenstackCommandProperties.REGIONS_DESCRIPTION
+  )
+  private List<String> regions = new ArrayList<>();
+
+  @Parameter(
+      names = "--add-region",
+      description = "Add this region to the list of managed regions."
+  )
+  private String addRegion;
+
+  @Parameter(
+      names = "--remove-region",
+      description = "Remove this region from the list of managed regions."
+  )
+  private String removeRegion;
+
+  @Parameter(
+      names = "--insecure",
+      description = OpenstackCommandProperties.INSECURE_DESCRIPTION
+  )
+  private Boolean insecure;
+
+  @Parameter(
+      names = "--user-data-file",
```

Comment:
Ah sorry forgot about this: Could you apply this converter shown [here](https://github.com/spinnaker/halyard/pull/188) to all files you pass as paths to the CLI? The linked issue gives some insight.